docs/resources/dashboard.md

```diff
@@ -11013,6 +11013,7 @@ Required:
 
 Optional:
 
+- **available_values** (List of String) The list of values that the template variable drop-down is be limited to
```

Comment:
```suggestion
- **available_values** (List of strings) The list of values that the template variable drop-down is be limited to.
```

Reply:
I'm afraid this one was autogenerated and beyond my control 😅